tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -511,12 +513,27 @@ const reducer = createReducer(
         }
       }
 
-      const resolvedResult = buildOrReturnStateWithUnresolvedImportedPins(
-        state.unresolvedImportedPinnedCards,
-        newCardIds,
+      // Updates pinned/original card id mapping and cardmetadatamap because
```

Comment:
On second thought, if you make changes to the documentation and API that I describe, then you probably don't need this comment.